Automattic/wp-calypso
# Remove More Informative Scan feature flag

#### Changes proposed in this Pull Request

* Since the More Informative Scan project was released we have no need for the feature flag anymore.

#### Testing instructions



* Set up a test site with a scan subscription
* Open up the scan pages and verify everything still looks good.
